crates/bin/prove_block/src/rpc_utils.rs

```diff
@@ -124,11 +125,8 @@ pub(crate) async fn get_storage_proofs(
     let mut storage_proofs = HashMap::new();
 
     log::info!("Contracts we're fetching proofs for:");
-    for contract_address in accessed_keys_by_address.keys() {
-        log::info!("    {}", contract_address.to_string());
-    }
-
     for (contract_address, storage_keys) in accessed_keys_by_address {
+        log::info!("    {}", contract_address.to_string());
```

Comment:
I know this was just moved, but I suggest giving some more context in the log message -- I had a hard time finding this log statement last time I wanted to remove it ;)